test/unit/query-generator/show-tables.query.test.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  expectsql,
+  createSequelizeInstance,
+} from '../../support';
+
```

Comment:
file should be named `show-tables-query.test.ts`

Reply:
https://github.com/sequelize/sequelize/pull/15101/commits/9ad25365e4198838f242f99c19416c55d424bc3d